news/2018-05-02-CWC-2018-registrations-open.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Registrations now open!"
+date: 2018-05-02 01:00:00 +0000
+---
+ 
+Fruits dropping, catchers dashing and commentators making _juicy_ puns on every single match. Here comes this time of the year again as the osu!catch World Cup 2018 registration phase begins!
+
+![](/wiki/shared/news/banners/CWC_2018_banner.jpg)
+ 
+For more details, refer to the **[osu!catch World Cup 2018 wiki article](/wiki/Tournaments/CWC/2018/)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/739799)**.
+ 
+## Tournament Schedule
+ 
++ Registration Phase: 2018-05-02/2018-05-16
++ Drawings: 2018-06-02 (14:00 UTC)
++ Group Stage: 2018-06-09/2018-06-10
++ Round of 16: 2018-06-16/2018-06-17
++ Quarterfinals: 2018-06-23/2018-06-24
++ Semifinals: 2018-06-30/2018-07-01
++ Finals: 2018-07-07/2018-07-08
++ Grand Finals: 2018-07-14/2018-07-15
+ 
+## Prizes
++ **1st:** Profile badge, "osu!catch Champion" user title for one year
++ **2nd:** Profile badge
++ **3rd:** Profile badge
+
+_The rest of the prizes will be determined during the registration phase._
+ 
+## How do I register?
+ 
++ **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/15)**
++ Do not create teams in advance! Teams will be formed by the country's team captain after the registration phase has concluded.
++ Hand in your registration before the **16th of May.**
```

Comment:
16th of May -> 2018-05-16

Reply:
I would not change this just to have a clear date for people.
(Also I might be wrong but ASC should apply to wiki articles and not news articles?)

Reply:
🤷‍♂️

Reply:
Leaving that as-is is ok, but the dates for the tournament schedule above should probably be changed to be similar to this then... maybe?

Reply:
not that big of a deal really, it is just hard to read the demanding format